Resources/views/Block/block_admin_list.html.twig

```diff
@@ -27,40 +27,16 @@ file that was distributed with this source code.
-                                                            {% if admin.subClasses is empty %}
-                                                                <a class="btn btn-link btn-flat" href="{{ admin.generateUrl('create')}}">
-                                                                    <i class="fa fa-plus-circle"></i>
-                                                                    {% trans from 'SonataAdminBundle' %}link_add{% endtrans %}
-                                                                </a>
-                                                            {% else %}
-                                                                <a class="btn btn-link btn-flat dropdown-toggle" data-toggle="dropdown" href="#">
-                                                                    <i class="fa fa-plus-circle"></i>
-                                                                    {% trans from 'SonataAdminBundle' %}link_add{% endtrans %}
-                                                                    <span class="caret"></span>
-                                                                </a>
-                                                                <ul class="dropdown-menu">
-                                                                    {% for subclass in admin.subclasses|keys %}
-                                                                        <li>
-                                                                            <a href="{{ admin.generateUrl('create', {'subclass': subclass}) }}">{{ subclass|trans({}, admin.translationdomain) }}</a>
-                                                                        </li>
-                                                                    {% endfor %}
-                                                                </ul>
-                                                            {% endif %}
-                                                        {% endif %}
-                                                        {% if admin.hasroute('list') and admin.isGranted('LIST') %}
-                                                            <a class="btn btn-link btn-flat" href="{{ admin.generateUrl('list')}}">
-                                                                <i class="fa fa-list"></i>
-                                                                {% trans from 'SonataAdminBundle' %}link_list{% endtrans -%}
-                                                            </a>
-                                                        {% endif %}
+                                                        {% for action in admin.dashboardActions %}
+                                                            {% include action.template|default('SonataAdminBundle:CRUD:dashboard__action.html.twig') with {'action': action} %}
```

Comment:
The issue you are referring to is still open, so no decision has been made yet. {% include %} is used everywhere in the code so far, so I think it might be better keeping it for now. If and when it is deprecated in Twig, it could be replaced everywhere in a single commit.